optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/roster/RosterState.java

```diff
@@ -42,7 +42,7 @@
     @NotNull
     private LocalDate firstDraftDate;
     @NotNull
-    private final Integer publishLength; // In number of days
+    private Integer publishLength; // In number of days
```

Comment:
Did Quarkus require this and Spring Boot somehow didn't, or did the change slip in from some other PR?

Reply:
Quarkus requires this for database deserialization